go.mod

```diff
@@ -1,132 +1,143 @@
 module github.com/openshift/csi-operator
 
-go 1.23.4
+go 1.24.4
```

Comment:
this looks suspicious, I would expect 
```
go 1.24.0

toolchain go1.24.4
```

Reply:
This happens because github.com/aws/karpenter-provider-aws v1.6.0 [needs](https://github.com/aws/karpenter-provider-aws/blob/main/go.mod) go version 1.24.4